src/mlpack/core/data/split_data.hpp

```diff
@@ -47,13 +48,15 @@ namespace data {
  * @param shuffleData If true, the sample order is shuffled; otherwise, each
  *       sample is visited in linear order. (Default true.)
  */
-template<typename T, typename U>
+template <typename T, class labelType,
```

Comment:
```suggestion
template <typename T, class LabelType,
```
For template parameters, we typically use uppercase letters to denote that it's a type instead of a variable.